integration-tests/steps/stack_groups.py

```diff
@@ -266,6 +277,12 @@ def step_impl(context, first_stack, second_stack):
     assert creation_times[stacks[0]] < creation_times[stacks[1]]
 
 
+@then('stack_group drift status is "{desired_status0}" and "{desired_status1}"')
+def step_impl(context, desired_status0, desired_status1):
+    assert context.output[0]["StackDriftStatus"] == desired_status0
+    assert context.output[1]["StackDriftStatus"] == desired_status1
```

Comment:
I'll also note that the way you have this test is basically assuming the dictionary values on `context.output` will always be in the same order. But I don't think you actually have that guarantee.